gradle/osgi.gradle

```diff
@@ -9,14 +9,13 @@ afterEvaluate {
 
             name = 'Mockito Mock Library for Java. Core bundle requires Hamcrest-core and Objenesis.'
             symbolicName = 'org.mockito.mockito-core'
-            version = project.version
+            version = project.version.replace('-', '.')
```

Comment:
It does not seem to be correct that the version of the bundle is different than the version of the jar (pom.xml). What is the consequence of this?